### PR TITLE
Setting up Build2019 Demo Sample Structure

### PR DESCRIPTION
**Pull request by andreiborodin**
This PR does the following:
- Creates a sample directory, and puts a Build2019Demo.Unity folder with a Unity project in there
- Takes MRTK 2.RC2.1 package into that
- Adds SpectatorView through a symlink

For symlinks to work in your git, you may need to set the following flag in the repo:
"git config core.symlinks true"
or for all repos:
"git config --global core.symlinks true"

### CODE REVIEW

File: samples/Build2019Demo.Unity/ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+m_EditorVersion: 2018.3.11f1
```
Comment (matthejo) [inline samples/Build2019Demo.Unity/ProjectSettings/ProjectVersion.txt]: >2018.3.11f1 [](start = 17, length = 11)

I think the other project is on 2018.3.14f1 currently, might be nice to keep these in sync

File: samples/Build2019Demo.Unity/Assets/MR.SpectatorView [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../src/SpectatorView.Unity/Assets
```
Comment (chrisfromwork) [inline samples/Build2019Demo.Unity/Assets/MR.SpectatorView]: nit: lets continue to name things MixedReality-SpectatorView

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Git Tools
+- Visual Studio
+- Unity 2018.3.14f1
+
+Once you have the required software, follow these steps:
+
+1. Clone this repository if you haven't yet.
+2. Using your favorite Git management tool, pull latest version of the code.
+3. Run /tools/scripts/ResetSamples.bat to get to a clean state of the Samples folder with appropriate configuration.
```
Comment (chrisfromwork) [inline samples/README.md]: nit: Please add more context around what this bat script does.

File: samples/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Ensure you have all the required software, which is the same as for the primary SpectatorView project:
+
+- Git Tools
+- Visual Studio
+- Unity 2018.3.14f1
+
+Once you have the required software, follow these steps:
+
+1. Clone this repository if you haven't yet.
+2. Using your favorite Git management tool, pull latest version of the code.
```
Comment (chrisfromwork) [inline samples/README.md]: nit: please call out that this requires the submodule flag

**Pull request merged**